LICENSE

```diff
@@ -174,5 +176,36 @@
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
 
-   END OF TERMS AND CONDITIONS
-
+Some portions of Thrust may be licensed under other compatible open-source
```

Comment:
It would be good to summarize which portions (the BSL notice in `NOTICE` does this).